Civilla-Project/Police-Control/src/main/java/CivillaAnalisysResponse.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Hashtable;
+
+public class CivillaAnalisysResponse {
+    public int status;
+    public String response;
+    public Hashtable<String, Float> content;
```

Comment:
The response got only floats? Or it could be expanded for various types?

Reply:
Yep. It is a simple service which should return crime coeficients only. What would you advice if I want it to be extendable?

Reply:
Store data as strings mayb
In the case of use only for return one coef mayb use only one float?